docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -2077,6 +2077,157 @@ Number of incompatible differences between published cluster states.
 (integer)
 Number of compatible differences between published cluster states.
 =======
+
+`master_timing`::
+(object)
+Contains low-level statistics about how long various activities took while the
+node was the elected master. Omitted if the node is not master-eligible. Every
+field whose name ends in `_time` within this object is also represented as a
+raw number of milliseconds in a field whose name ends in `_time_millis`. The
+human-readable fields with a `_time` suffix are only returned if requested with
+the `?human=true` query parameter.
++
+.Properties of `master_timing`
+[%collapsible]
+=======
+`unchanged`::
+(object)
+Contains statistics about cluster state update attempts that did not change the
+cluster state.
++
+.Properties of `unchanged`
+[%collapsible]
+========
+`count`::
+(long)
+The number of cluster state update attempts that did not change the cluster
+state since the node started.
+
+`computation_time`::
+(<<time-units,time value>>)
+The cumulative amount of time spent computing no-op cluster state updates since
+the node started.
+
+`notification_time`::
+(<<time-units,time value>>)
+The cumulative amount of time spent notifying listeners of a no-op cluster
+state update since the node started.
+
+========
+
+`success`::
+(object)
+Contains statistics about cluster state update attempts that did not change the
```

Comment:
I think these are update attempts that *did* change the cluster state successfully?

Reply:
Bah yes. Copy/paste might have been involved a bit here.